docs/src/terminology.md

```diff
@@ -220,6 +228,10 @@ A computer participating in a [cluster](#cluster).
 
 The number of [validators](#validator) participating in a [cluster](#cluster).
 
+## optimistically confirmed block
+A [block](#block) which got the [votes](#vote) from [validators](#validator)
+holding more than 2/3 of total [stake](#stake) in a [cluster](#cluster).
```

Comment:
i think we need to mention that the votes considered for optimistic confirmation may not have been committed to a block yet.

Reply:
Added. But I suddenly find there is a definition of "confirmed block" above, should I just mark that "we sometimes call it optimistically confirmed"?

Reply:
maybe just change the `confirmed block` definition to be something like this, given that it is the less precise term?
```
see: [optimistically confirmed block](#optimistically-confirmed-block)
```